model/qti/Element.php

```diff
@@ -751,6 +755,14 @@ protected function getArraySerializedPrimitiveCollection(array $elements): array
         return $data;
     }
 
+    public function getServiceLocator()
```

Comment:
This code was was without the `ServiceLocator` for a long time. I am not sure the solution should be here instead of adding `setServiceLocator` in https://github.com/oat-sa/extension-tao-mediamanager/blob/71b6784f951ef5de1e75c01afc19125b55db1660/model/sharedStimulus/parser/JsonQtiAttributeParser.php

Now we are calling static `ServiceManager` in one more place

Reply:
Ok. Now I understood. The serviceLocator trait was added to this class and it is required now. @bartlomiejmarszal  created a technical debt to remove this from here.